### PR TITLE
Add ADRs about using root user and database volumes in Breeze

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
cc: @Bowrna

**Comment by potiuk** [comment]
Would love a review on that one - it only documents decisions for CI/Breeze (and I am going add few more soon).

File: dev/breeze/doc/adr/0006-using-root-user-and-fixing-ownership-for-ci-container.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+On MacOS and Windows this is not needed. Both MacOS and Windows use
+"user-space" filesystems to mount files. The filesystems are far slower than
+the native filesystem (many times actually) - which impacts the speed of
+running Airflow in Docker container on MacOS and Windows. However, they
+automatically remap the user - all the files created inside the containers are
+automatically remapped to have the "host" user ownership and there is no
+need to fix the ownership for those cases.
+
+On Linux any file we create in container will keep the same user is and group
+id in the host. But those user/group ids might not exist in the Host. - if we
```
Comment (eladkal) [inline dev/breeze/doc/adr/0006-using-root-user-and-fixing-ownership-for-ci-container.md]:
```suggestion
On Linux any file we create in container will keep the same user id and group
id in the host. But those user/group ids might not exist in the Host. - if we
```

File: dev/breeze/doc/adr/0006-using-root-user-and-fixing-ownership-for-ci-container.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+it finds all files owned by "root" in those directories and changes their
+ownership to be `HOST_USER/HOST_GROUP`. This way when you exit the
+containers on linux, the files are owned by the host user, and can be
+easily deleted - either manually or when you change branches.
+
+
+## Alternatives
+
+The problem could potentially be mitigated by
+[user remapping]( https://docs.docker.com/engine/security/userns-remapi/)
```
Comment (eladkal) [inline dev/breeze/doc/adr/0006-using-root-user-and-fixing-ownership-for-ci-container.md]: This link leads to nowhere...
Did you mean?
```suggestion
[user remapping]( https://docs.docker.com/engine/security/userns-remap/)
```

**Review by eladkal — APPROVED** [review]

File: dev/breeze/doc/adr/0006-using-root-user-and-fixing-ownership-for-ci-container.md (hunk 2 above)
Comment (potiuk) [reply dev/breeze/doc/adr/0006-using-root-user-and-fixing-ownership-for-ci-container.md]: Guess what editor I used to edit this file ;)

```
i - insert before the cursor
```

**Pull request merged**